backend/src/apiserver/client_manager.go

```diff
@@ -214,6 +214,11 @@ func initDBClient(initConnectionTimeout time.Duration) *storage.DB {
 		glog.Fatalf("Failed to initialize the databases.")
 	}
 
+	response = db.Model(&model.Experiment{}).RemoveIndex("Name")
```

Comment:
What's DB schema before this command?
e.g. Auto migration is done, but this Name index hasn't been dropped.

Reply:
Also, shall we consider defaulting to current POD_NAMESPACE?

Reply:
> What's DB schema before this command?
> e.g. Auto migration is done, but this Name index hasn't been dropped.

The table has the additional column `Namespace` and an index on the (`Name`, `Namespace`) pair. It still has the index on `Name`.
Per http://gorm.io/docs/migration.html#Auto-Migration, AutoMigrate only makes additions but not subtractions.

> Also, shall we consider defaulting to current POD_NAMESPACE?

We will make the namespace required when user creates experiment, so a default value won't be necessary. For existing experiments though, I think the shared (playground) namespace might be a better choice for backfill.

Reply:
Thanks! SGTM